anomaliesDetection/anomalies.go

```diff
@@ -77,16 +76,24 @@ type (
 )
 
 // RunAnomaliesDetection run every anomaly detection algorithms and store results in ElasticSearch.
-func RunAnomaliesDetection(account aws.AwsAccount, ctx context.Context, tx *sql.Tx) error {
-	now := time.Now()
-	today := time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, time.UTC)
+func RunAnomaliesDetection(account aws.AwsAccount, lastUpdate time.Time, ctx context.Context) (time.Time, error) {
+	now := time.Now().UTC()
+	today := time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, now.Location())
+	begin := lastUpdate.Add(-1 * time.Hour * time.Duration(30*24))
```

Comment:
If lastUpdate is the very minimum of a time.Time, can we still remove 30 days?

Reply:
yes, it works